### ISSUE TITLE
New profile context menu states: send message, add contact, rename

### ISSUE THREAD

**Issue by iurimatias** [status-im/status-desktop#6113]
### Description

needs https://github.com/status-im/status-desktop/issues/6125

https://www.figma.com/file/idUoxN7OIW2Jpp3PMJ1Rl8/%E2%9A%99%EF%B8%8F-Settings-%7C-Desktop?node-id=1125%3A139397

In this task the goal is to add the options for Send Message, Send Contact Request, and Rename.
The Identity related options are out of scope for this task

### Acceptance Criteria

- [x] If the profile is a mutual contact, it should have the `Send Message` option

![Screen Shot 2022-06-16 at 10 06 22 AM](https://user-images.githubusercontent.com/176720/174088467-f90cf258-4b7e-48f5-8155-9b2db23113c7.png)

- [x] If the profile is NOT a mutual contact and is NOT blocked AND there is no pending contact request it should display the `Send Contact Request` option

![Screen Shot 2022-06-16 at 10 14 44 AM](https://user-images.githubusercontent.com/176720/174089784-70eba1b0-d139-4302-bdd3-fc90b2d65a14.png)

- [x] If the profile is blocked it should NOT show either `Send Message` or `Send Contact Request` and should show `Unblock User`
  
![Screen Shot 2022-06-16 at 10 15 40 AM](https://user-images.githubusercontent.com/176720/174090070-b9b052f3-9872-4f62-923f-381a5d6506a7.png)

- [x] Show option for `Rename` which should display the current alias/rename modal

**Comment by MishkaRogachev** [comment]
@iurimatias should I also add `Block User` and what are the conditions?
